package.json

```diff
@@ -58,6 +58,7 @@
     "array-includes": "^3.1.6",
     "awesome-debounce-promise": "^2.1.0",
     "bootstrap-filestyle": "~1.2.1",
+    "bootstrap-select": "^1.13.18",
```

Comment:
Do we need this line up here, or is the resolution sufficient?

Reply:
We can add it if we want but it won't actually do anything. Yarn would use the package in the resolutions.

Reply:
I'm suggesting to remove it up here, unless we actually depend on it as a primary dependency. Packages up here should only be the primary dependencies we care about, and then resolutions should be used to "fix" transitive dependencies.

Reply:
@Fryguy I tried it without this line and bin/update doesn't work so I think we do need it